policy/modules/system/userdomain.if

```diff
@@ -937,6 +937,10 @@ template(`userdom_common_user_template',`
 		virt_home_filetrans_svirt_home($1_t, dir, "qemu")
 		virt_home_filetrans_virt_home($1_t, dir, "VirtualMachines")
 	')
+
+	optional_policy(`
+		tpm2_run($1_t, $1_r)
```

Comment:
Why does this need to be run by unprivileged user domains?

Reply:
I think TPM provides general capabilities, not that tools only privileged users can use. Of course, this is not necessary. If it is not necessary, I will remove this line.

Reply:
I'd prefer to remove this since there is no specific need.

Reply:
Got it, updated already.